BossMod/Modules/Shadowbringers/Foray/Duel/Duel2Lyon/Duel2Lyon.cs

```diff
@@ -0,0 +1,17 @@
+namespace BossMod.Shadowbringers.Foray.Duel.Duel2Lyon
```

Comment:
Please move that to a different PR, since it's a work in progress - let's focus on what is already done here.

Reply:
done